### DISCUSSION TITLE
SED Plotting

### DISCUSSION THREAD

**Discussion by olliepye** [mboquien/cigale#98] [Q&A] [answered]
Hello,

I just have a question RE the SED plotting. I have some plots where the emission from a given process (e.g. AGN) is greater than the best fit line. My understanding was that all of the emission types 'added' together to give the final best fit line. Can you please assist with my understanding here?
Here is an example: 
[21383_best_model.pdf](https://github.com/user-attachments/files/16120962/21383_best_model.pdf)

Another example (taken from the Cigale Manual). The AGN emission is just above the best fit at $\lambda \approx 10^{-1}\mu$ m.

![20240708_08h20m23s_grim](https://github.com/mboquien/cigale/assets/42687811/ef9e6fba-9447-423b-a874-e810d885fee5)

Thanks,
Ollie

**Comment by mboquien** [comment, marked as answer]
Indeed, it should be the sum of all components. The reason of the difference, however, is that the model spectrum accounts for the IGM absorption, which is not the case for individual components. I should correct this at some point.

**Reply by olliepye** [reply]
Thank you for clarifying!
